<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showWave</class>
 <widget class="QMainWindow" name="showWave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxBtn">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>20</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>operation</string>
    </property>
    <widget class="QPushButton" name="showWaveBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示波形</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseWaveBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停波形</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearWaveBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除波形</string>
     </property>
    </widget>
    <widget class="QPushButton" name="integralBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>积分</string>
     </property>
    </widget>
    <widget class="QPushButton" name="differBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>微分</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>650</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;x轴/(10*pixel)：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y轴/(10*pixel):</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>210</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>other</string>
    </property>
    <widget class="QPushButton" name="drawGridBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示网格</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="gridNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveWaveBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存波形</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadFileBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showNormalBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示原波形</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="NValueSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>650</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="slideBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>650</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>滑动均值滤波法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="midAvgBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>650</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>中位值平均滤波法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>650</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>加权递推平均滤波算法</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="interalPixelsSpin">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>650</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>650</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>每隔N个像素显示一个点：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>650</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设置曲线宽度：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="penWidthSpin">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>650</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>650</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设置画另一曲线宽度：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="otherPenWidthSpin">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>650</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showMathAvgBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>平均值滤波</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1374</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
